app/models/miq_ae_yaml_import.rb

```diff
@@ -310,10 +310,10 @@ def new_domain_name_valid?
     true
   end
 
-  def update_attributes(domain_obj)
+  def update(domain_obj)
```

Comment:
All these changes are pretty straightforward replacements.  Changing this method name is the only one that has potentially more exposure.

@mkanoor Do you know if this method would be called from any other repo where we might need to make the corresponding name change?

Reply:
Oh that was me being an idiot, sorry, it was late. This shouldn't be here.

Reply:
Maybe/maybe not.  It is likely this name was picked to match the Rails method, so changing it to match may be the right thing.  Let's give @mkanoor  some time to respond before changing.

Reply:
Confirmed that https://github.com/ManageIQ/manageiq-automation_engine/pull/367#pullrequestreview-288041339 was in response to my question here.  So we are good with the method name change.

Thanks @d-m-u